### PR TITLE
Fix quoting of MySQL table name

### PR DESCRIPTION
**Pull request by cpcloud**
Fixes #316

### CODE REVIEW

**Comment by cpcloud** [comment]
@llllllllll any comments here? else merging on pass

File: odo/backends/sql_csv.py [hunk 1]
```diff
@@ -84,65 +84,97 @@ def compile_from_csv_sqlite(element, compiler, **kwargs):
...
-    assert not stdout, \
-        'error: %s from command: %s' % (stdout, ' '.join(cmd))
+    stdout, stderr = subprocess.Popen(
+        cmd,
+        stdout=subprocess.PIPE,
+        stderr=subprocess.STDOUT,
+        stdin=subprocess.PIPE
+    ).communicate()
+    if stdout:
+        raise RuntimeError(
```
Comment (llllllllll) [inline odo/backends/sql_csv.py]: Could we raise something more specific here, I normally think of runtime error as recursion depth and notimplemented style issues.

**Comment by llllllllll** [comment]
Other than the one comment this looks good. I can close the other pr I have open about encodings also.

File: odo/backends/sql_csv.py (hunk 1 above)
Comment (cpcloud) [inline odo/backends/sql_csv.py]: Yeah I'll raise a sqlalchemy exception here instead.

Comment (llllllllll) [inline odo/backends/sql_csv.py]: thanks

**Pull request merged**